protected/components/GuideNavigation.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
this reusable component replaces the guidelines navigation for each guidelines page. I attempted to separate view from logic